07,10CLASSWORKTASK2/07,10CLASSWORKTASK2/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace _07_10CLASSWORKTASK2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var m = Input("m");
+            var n = Input("n");
+            Console.WriteLine("Числа больше 10:" + L(m, n));
+            Console.ReadKey();
+        }
+
+        static bool L(double m, double n)
+        {
+            return m > 10 & n > 10;
```

Comment:
Рекомендую использовать ленивую коеъюнкцию
```suggestion
            return m > 10 && n > 10;
```